test/k8sT/Policies.go

```diff
@@ -454,6 +456,97 @@ var _ = Describe("K8sValidatedPolicyTest", func() {
 				res.ExpectSuccess("Egress DNS connectivity should be allowed for pod %q", pod)
 			}
 		})
+
+		It("Allows traffic with k8s default-allowingress policy", func() {
+
+			res := kubectl.ExecPodCmd(
+				helpers.DefaultNamespace, appPods[helpers.App2],
+				helpers.CurlFail(fmt.Sprintf("http://%s/public", clusterIP)))
```

Comment:
could you change to `elpers.CurlFail("http://%s/public", clusterIP))`?

+ the other ones added ;-)